run_local.sh

```diff
@@ -1,13 +1,51 @@
 #!/bin/bash
+
+python -c 'import configurations' 2>&1>/dev/null
```

Comment:
This stuff makes the `-ex` output look super messy, but it helps identify issues like not having Python dependencies installed for new developers.

Reply:
I dig it, I like ex for debugging but that's like hella optional.